staging/src/k8s.io/apiserver/pkg/audit/request.go

```diff
@@ -140,6 +130,29 @@ func LogRequestObject(ae *audit.Event, obj runtime.Object, gv schema.GroupVersio
 			ae.ObjectRef.ResourceVersion = meta.GetResourceVersion()
 		}
 	}
+	// TODO: ObjectRef should include the API group.
+	if gvr.Version != "" {
```

Comment:
Done. I think both are used equally in Kubernetes, but I agree that `len` is the convention here.